<launch>
    <node pkg="rr_platform" type="image_transform" name="image_transform" output="screen">
        <param name="transform_topics" value="/canny_color_edges"/>
    </node>

    <node pkg="rr_common" type="image_pcl_converter" name="image_pcl_converter" output="screen">
        <param name="image_topics" value="/canny_color_edges_transformed"/>
        <param name="px_per_meter" value="100.0"/>
    </node>
</launch>
